### DISCUSSION TITLE
How to use wildcards in "My Filters"

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [uBlockOrigin/uBlock-issues#2416] [Q&A] [answered]
mangatoki keeps adding a number of increasing value to their urls, such as mangatoki167.net,mangatoki168.net,etc. And then I have to create a new filter to remove all the ad banners which should be easy, as they're all loaded from "mangatoki*.net/bbs/linkbn.php," except that's irritating and stressful. But for some reason ublock doesn't do anything when I added `mangatoki*.net##[href^="https://mangatoki*.net/bbs/linkbn.php"]` as a filter. It also doesn't work with 
`mangatoki|*|.net##[href^="https://mangatoki|*|.net/bbs/linkbn.php"]`, which I tried referencing the only page in this repo that came up when I searched for "wildcard..." Please tell me ublock supports URL wildcards for (user) cosmetic filtering...

**Comment by Yuki2718** [comment, marked as answer]
Entity filter won't work. Why don't add a [generic cosmetic filter](https://help.adblockplus.org/hc/en-us/articles/360062733293#generic-specific) `##[href^="https://mangatoki"][href*=".net/bbs/linkbn.php"]`? (BTW what is the current number? `mangatoki168.net` is dead.)

**Reply by neurodiverseEsoteric** [reply]
I believe it's at 167 actually. But the generic cosmetic filter you shared didn't work either...

**Reply by Yuki2718** [reply]
I can't access 167 either, from which country/region do you access? The rule is entirely based on your description. Make sure you didn't add domain to the rule - it must look like

![rule](https://user-images.githubusercontent.com/58900598/209091747-9b13eb78-4453-4a01-8c2f-edcf73e3f4d3.png)

and also didn't check "Ignore generic cosmetic filters". But it's much better to provide *exact and full* URL with screenshot(s) of what you wanna hide, the problem would have already been solved if you did this.

**Reply by neurodiverseEsoteric** [reply]
MY APOLOGIES. The problem was that there is no 'g.' It's MANATOKI, not manGatoki. This is something I thought blue light filters, opendyslexic, glasses, and adderall were supposed to fix. With the typo eradicated, the filter works. Thanks and again sorry about the misunderstanding...

**Reply by Yuki2718** [reply]
That is already blocked by List-KR with a regex rule.

**Reply by neurodiverseEsoteric** [reply]
Ah...I stopped usin the regional filters yeard ago since they messed up sites that the general lists hadn't...

**Comment by uBlock-user** [comment]
Use regex if you also need a network filter for mangatoki* sites.
